api/src/controllers/communityUserController.js

```diff
@@ -101,6 +101,54 @@ const getAllMembers = async (req, res) => {
   }
 }
 
+// @desc Get the community-users
+// @route GET /api/community-users/community/:id/details
+// @access Public
+
+const getAllMemberDetails = async (req, res) => {
```

Comment:
seems like getAllMemberDetails, getAllMembers, getCommunityUsers can be single function